src/views/mfa-verify/InlineTOTPForm.js

```diff
@@ -31,6 +31,7 @@ define(['okta', 'views/shared/TextBox'], function (Okta, TextBox) {
         if (!form.isValid()) {
           return;
         }
+        form.options.appState.set('trapMfaRequiredResponse', true);
```

Comment:
I think we should trap the response only if we are in MFA_CHALLENGE because of the ```previous``` call we need to make. 
If we are in MFA_REQUIRED and we type the code and press Verify, that will make a request to the backend, and if the response is MFA_REQUIRED again, we should let RouterUtil do its magic.